operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md

```diff
@@ -110,9 +113,24 @@ Use the `cray hsm locks lock` command to perform locking.
    [Success]
    ComponentIDs = [ "x3000c0s5b0n0", "x3000c0s4b0n0", "x3000c0s7b0n0", "x3000c0s6b0n0", "x3000c0s3b0n0", "x3000c0s2b0n0", "x3000c0s9b0n0", "x3000c0s8b0n0",]
    ```
+   
+1. Lock the NodeBMCs of those management nodes
+   ```bash
+   ncn# cray hsm locks lock create --component-ids $(cray hsm state components list --role management --type node --format json | jq '.Components[].ID' | sed 's/n[0-9]*//;s/"//g' | tr '\n' ',' | sed 's/.$//')
```

Comment:
Separate command input/output here and in other steps below where it is in one code block. Here is an example:

```
ncn# cray hsm locks lock create --component-ids $(cray hsm state components list --role management --type node --format json | jq '.Components[].ID' | sed 's/n[0-9]*//;s/"//g' | tr '\n' ',' | sed 's/.$//')
```


Example output:


```
Failure = []
[Counts]
Total = 8
Success = 8
Failure = 0
[Success]
ComponentIDs = [ "x3000c0s5b0", "x3000c0s4b0", "x3000c0s7b0", "x3000c0s6b0", "x3000c0s3b0", "x3000c0s2b0", "x3000c0s9b0", "x3000c0s8b0",]
```